scout/server/app.py

```diff
@@ -173,6 +176,25 @@ def cosmic_prefix(cosmicId):
         return cosmicId
 
 
+def configure_oauth_login(app):
+    """Register the Google Oauth login client using config settings"""
+
+    google_conf = app.config["GOOGLE"]
+    discovery_url = google_conf.get("discovery_url")
+    client_id = google_conf.get("client_id")
+    client_secret = google_conf.get("client_secret")
+
+    extensions.oauth_client.init_app(app)
+
+    extensions.oauth_client.register(
+        name="google",
+        server_metadata_url=discovery_url,
+        client_id=client_id,
+        client_secret=client_secret,
+        client_kwargs={"scope": "openid email profile"},
```

Comment:
No need of all those all params any more, since we are using openid!

Reply:
🥇